db/migrate/20230608133616_add_url_slug_to_branch.rb

```diff
@@ -0,0 +1,5 @@
+class AddUrlSlugToBranch < ActiveRecord::Migration[7.0]
+  def change
+    add_column :branches, :url_slug, :string
```

Comment:
Add a script to update old branches data

Reply:
Done

Reply:
Done